src/app/components/Navbar.tsx

```diff
@@ -0,0 +1,14 @@
+
+const Navbar = () => {
+    return (
+        <div className="flex flex-row w-full items-center justify-center gap-[10vw] p-[1em] border-t border-b border-1 border-black">
+                <p className="text-xl">Our Story</p>
+                <p className="text-xl">Link</p>
+                <h1 className="text-3xl text-main-pink text-bold font-bold">Swapify</h1>
+                <p className="text-xl">Login</p>
+                <p className="text-xl">Messages</p>
+        </div>
```

Comment:
Not actually important, but the big websites use `<nav>` to wrap the navbar instead of div. Functionally its the same